hnclient-v2/src/main/java/ca/bc/gov/hlth/hnclientv2/HnsClientMainMethod.java

```diff
@@ -5,7 +5,7 @@
 /**
  * Main class that boots the Camel application.
  */
-public class MainMethod {
+public class HnsClientMainMethod {
```

Comment:
Need to update the class name in the camel-maven-plugin in the pom also.

Reply:
Fixed in ce24d4153daacf73c6f52ccfbfec86da003b65ca.